Dockerfile

```diff
@@ -90,6 +90,13 @@ RUN if [ "$AUTO_SEGMENTATION" = "yes" ]; then \
     bash -i /tmp/components/auto_segmentation/install.sh; \
     fi
 
+ARG TRACKING
+ENV TRACKING=${TRACKING}
+ENV TRACKING_PATH=${HOME}/tracking
+RUN if [ "$TRACKING" = "yes" ]; then \
+    bash -i /tmp/components/tracking/install.sh; \
```

Comment:
Tracking may be used without GOTURN.
Consider making using of GOTURN model optional.